<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Select_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>串口选择：</string>
    </property>
   </widget>
   <widget class="QLabel" name="BaudRate_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>68</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>波特率：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Select_Box">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="BaudRate_Box">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Operate_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>串口操作</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Operate_Button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="DisplayTXT_Brower">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>431</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="SerialInput0_Edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>461</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SerialSend_Button">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendClear_Button">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清除发送</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
